### PR TITLE
Update eslint-plugin-react + README to track latest

### PR DESCRIPTION
**Pull request by baer**
@ryan-roemer

### CODE REVIEW

File: rules/react/off.js [hunk 1]
```diff
@@ -68,17 +31,68 @@ module.exports = {
...
+    // Enforce ES5 or ES6 class for returning value in render function
+    "require-render-return": 0,
     // Prevent extra closing tags for components without children
     "react/self-closing-comp": 0,
     // Enforce component methods order
     "react/sort-comp": 0,
+    // Enforce propTypes declarations alphabetical sorting
+    "sort-prop-types": 0,
     // Prevent missing parentheses around multilines JSX
     "react/wrap-multilines": 0
```
Comment (dylancwood) [inline rules/react/off.js]: Missing comma

File: rules/react/walmart.js [hunk 2]
```diff
@@ -69,17 +31,68 @@ module.exports = {
...
+    // Enforce ES5 or ES6 class for returning value in render function
+    "require-render-return": 0,
     // Prevent extra closing tags for components without children
     "react/self-closing-comp": 2,
     // Enforce component methods order
     "react/sort-comp": 0,
+    // Enforce propTypes declarations alphabetical sorting
+    "sort-prop-types": 0,
     // Prevent missing parentheses around multilines JSX
     "react/wrap-multilines": 2
```
Comment (dylancwood) [inline rules/react/walmart.js]: Missing comma

**Comment by baer** [comment]
Sorry bout that! How careless of me :). Should be fixed now.

**Comment by dylancwood** [comment]
:ship:
Probably good to get @ryan-roemer's eyes on it as well though

**Comment by ryan-roemer** [comment]
@baer @dylancwood -- LGTM. anyone get a change to test this on a real project?

**Comment by dylancwood** [comment]
@baer. I don't have any react projects laying around to test this on. 

Also, looks like a CLI merge is in order...

**Pull request merged**